src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -563,8 +447,9 @@ private String createChecksum(String input) {
         return sb.toString();
     }
 
+
     private String getJetzt_ddMMyyyy_HHmm() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)